schedule/models/calendars.py

```diff
@@ -104,46 +104,18 @@ def get_calendars_for_object(self, obj, distinction=''):
 
 
 @python_2_unicode_compatible
-class Calendar(models.Model):
+class AbstractCalendar(models.Model):
     '''
```

Comment:
I know it's not your line, but could you please replace ''' with """ in docstring?
https://www.python.org/dev/peps/pep-0008/#documentation-strings